public/app/plugins/datasource/cloud-monitoring/README.md

```diff
@@ -0,0 +1,7 @@
+# Google Cloud Monitoring Data Source - Native Plugin
+
+Grafana ships with built-in support for Google Cloud Monitoring. You just have to add it as a data source and you will be ready to build dashboards for your Cloud Monitoring metrics.
+
+Read more about it here:
+
+[http://docs.grafana.org/datasources/cloudmonitoring/](http://docs.grafana.org/datasources/cloudmonitoring/)
```

Comment:
```suggestion
[https://grafana.com/docs/grafana/latest/features/datasources/cloudmonitoring/](https://grafana.com/docs/grafana/latest/features/datasources/cloudmonitoring/)
```

Reply:
Also, I wonder if cloud monitoring have a different logo than stackdriver and if so are we suppose to change that?

Reply:
> Could you please open a PR for images so that we can verify that before merging this?

https://github.com/grafana/website/pull/1724